CMakeLists.txt

```diff
@@ -11,6 +11,7 @@ option(ENABLE_MEM_RTL "Enable Real-time library memory management functions (dis
 option(FATAL_GL_ERRORS "Check OpenGL calls and raise exceptions on errors." OFF)
 option(FORCE_COLORED_OUTPUT "Always produce ANSI-colored compiler warnings/errors (GCC/Clang only; esp. useful with Ninja)." OFF)
 option(FORCE_PORTABLE_INSTALL "Install all files into local directory defined by CMAKE_INSTALL_PREFIX" ON)
+option(USE_SYSTEM_PLOG "Use system plog library instead bundled" OFF)
```

Comment:
Do any distribution actually package it? It's header-only, so the package would not contain any shared library or anything.

Reply:
At least I saw plog library in vcpkg, and there other distributions can package it: https://repology.org/project/plog/versions

Reply:
it could be a good idea to use VCPKG then! So packagers only have to turn off `USE_VCPKG`, and we don't have to remind people to init submodules

Reply:
Enabled plog on vcpkg.

Reply:
So now there is no point keeping the `USE_SYSTEM_PLOG` option, people will set `USE_VCPKG` or not instead

Reply:
I think I get what you want to do:
1. If user has plog packaged in the distribution (only major distros with a plog package being Fedora/Gentoo/AUR), set `USE_SYSTEM_PLOG` and don't use the submodule
2. If user has `USE_VCPKG`, install it through VCPKG and ignore submodule
3. Otherwise, use submodule, ie for distributions that don't package it

Now that I think about it the logic is a little complicated. We have to cover 3. for distros such as Debian that cant use VCPKG and don't have a native plog package. Maybe VCPKG is not necessary after all...

Reply:
Actually there only 2 options - use system plog or use it from submodule. `USE_SYSTEM_PLOG` + vcpkg (if it enabled) makes plog system dependency, it's not important where it came from - we threats it as external library.